pf/internal/pfutils/proposed_new_test.go

```diff
@@ -165,7 +174,10 @@ func TestProposedNewBaseCases(t *testing.T) {
 		"null-unknown": {
 			Config: unk(),
 			Prior:  prim(nil),
-			Want:   object(map[string]valueBuilder{"foo": prim(nil)}),
+			// TODO[pulumi/pulumi-terraform-bridge#2247]: patching objchange.ProposedNew changed behavior
+			//
+			// Want:   object(map[string]valueBuilder{"foo": prim(nil)}),
+			Want: unk(),
```

Comment:
These all look "more correct" here in the absence of a planmodifier. Good to see we don't have any regressions with the objchange patch in pf